### PR TITLE
Fix typo in doc string

### PR DESCRIPTION
**Pull request by slipset**
Before submitting a PR make sure the following things have been done:

- [ ] The commits are consistent with our [contribution guidelines](CONTRIBUTING.md)
- [ ] You've added tests to cover your change(s)
- [ ] All tests are passing
- [ ] The new code is not generating reflection warnings

Keep in mind that new orchard builds are automatically deployed to Clojars
once a PR is merged, but **only** if the CI build is green.

Thanks!

### CODE REVIEW

File: src/orchard/apropos.clj [hunk 1]
```diff
@@ -34,7 +34,7 @@
      vars)))
 
 (defn find-symbols
-  "Takes a map and returns a list of maps containg map, doc and type.
+  "Takes a map and returns a list of maps containg name, doc and type.
```
Comment (bbatsov) [inline src/orchard/apropos.clj]: containing

File: src/orchard/apropos.clj [hunk 2]
```diff
@@ -34,7 +34,7 @@
...
   `:var-query` See `vars`.
   `:full-doc?` Causes the full doc to be returned instead of the abbreivated form.
```
Comment (bbatsov) [inline src/orchard/apropos.clj]: abbreviated

**Comment by slipset** [comment]
@bbatsov Will you do a squash-merge, or do you want me to rebase into one commit?

**Comment by bbatsov** [comment]
👍

**Pull request merged**